Client/Experiments/Messaging/NimbusMessagingHelperUtilityProtocol.swift

```diff
@@ -15,6 +15,7 @@ protocol NimbusMessagingHelperUtilityProtocol {
 /// Responsible for creating a ``NimbusMessagingHelper`` with appropriate context.
 class NimbusMessagingHelperUtility: NimbusMessagingHelperUtilityProtocol {
     private var logger: Logger
+    private let nimbus: NimbusMessagingProtocol = Experiments.shared
```

Comment:
This seems like it should probably be in the initializer with a default value, so if we want to eventually mock for testing, we can pass that in easily.